client/my-sites/plugins-wpcom/plugin-types/business-plugin.jsx

```diff
@@ -3,6 +3,17 @@ import noop from 'lodash/noop';
 
 import Gridicon from 'components/gridicon';
 
+/**
+ * Detect if the given url is a fully formed url
+ *
+ * @param {String} url - url to check
+ * @return {Boolean} True if it's a fully formed url
+ */
+
+const isFullyFormedURL = url => {
```

Comment:
this doesn't actually detect if we have a fully-formed URL, but I think it does what we need it to. could we change the name to something like, `hasHttpProtocol`